pkg/systemd/services/service-details.jsx

```diff
@@ -126,9 +127,9 @@ export class ServiceTemplate extends React.Component {
 
     render() {
         return (
-            <>
-                {this.state.error && <Alert variant="danger" isInline title={this.state.error} />}
-                <div className="panel panel-default">
+            <Card>
+                <CardBody>
+                    {this.state.error && <Alert variant="danger" isInline title={this.state.error} />}
```

Comment:
I am not sure if this alert should be in the card or outside of it.
on this pr:
![incard](https://user-images.githubusercontent.com/12330670/91961727-dc9e3380-ed0b-11ea-98d1-3c7f4115b90b.png)
moving it outside of the card:
![outsideofcard](https://user-images.githubusercontent.com/12330670/91961741-e32cab00-ed0b-11ea-88d8-13c2acdf6bff.png)

Also there is that line, not sure where it comes from :man_shrugging:

Reply:
@garrett 
I tend to believe to Alert should be out, but I leave it up to you.
(I will check the line)

Reply:
Not sure what you mean by "that line", but the strong red one on top of the alert box is part of the design: https://www.patternfly.org/v4/documentation/react/components/alert

Reply:
There is a thin line above the template name, inside the card, I just noticed as well.

Reply:
This is a really weird page. There's a random `<hr>` for no reason, some space below the button, no heading, and no guidance at all what someone is supposed to do here. Additionally, I think the input is unstyled.

The error should either be inside like the first item (which feels weird to me) or outside the card (treating it as a complete unit) and not be styled like an inline alert, but as a default one.

Reply:
I really don't understand what's supposed to be happening here at all. (And if I don't, then 98% of our users probably wouldn't either.)

Reply:
Perhaps we should just leave it inline, in the card, and come back to this later. It already has all sorts of odd issues. We could do a follow-up to attempt this page.

Reply:
ack thanks, so I am merging it as is.